trade-finance-manager-ui/server/controllers/utilisation-reports/index.ts

```diff
@@ -28,3 +29,35 @@ export const getUtilisationReports = async (req: Request, res: Response) => {
     return res.render('_partials/problem-with-service.njk', { user });
   }
 };
+
+export const getUtilisationReportByBankId = async (req: Request, res: Response) => {
+  const { userToken, user } = req.session;
+  const { bankId } = req.params;
+
+  const errors = validationResult(req);
```

Comment:
It doesn't look like this will ever return anything, since the validation errors are already handled (with a redirect) by the relevant validation middleware used for this endpoint (`validateUserTeam` and `validateBankId`).

Reply:
I'm not sure why, but when I first wrote this I don't remember that being the case. I also think I was trying to start encouraging the pattern of `res` logic being handled only in the controller, but if we're going to keep the portal pattern then I'm happy to do that

Reply:
(Seems that was a lie because I added the validation middleware...)